src/docs/build-deploy-and-maintain-apps/maintain-an-application.md

```diff
@@ -0,0 +1,498 @@
+
+```
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    name: sample-app
+    apiVersion: apps/v1
+  minReplicas: 3
+  maxReplicas: 5
+  metrics:
+    - type: Resource
+      resource:
+        name: memory
+        target:
+          type: AverageValue
+          averageValue: 1500Mi
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 70
+```
+When the average RAM utilization surpasses 1500Mi or the average CPU utilization goes beyond 70% of the maximum, the HPA will increase the deployment from three pods to four. 
+
+If the average utilization continues to rise above the threshold, a fifth pod will be created. However, when the average utilization remains below the threshold for several minutes, the HPA will gradually scale down the deployment. To avoid rapid scaling up and down of pods, there is a brief delay in the scaling-down process.
+
+**Check HPA status**
+
+Periodically check the status of your HPA with 'oc describe'.
+
+```
+$ oc describe hpa sample-app
+Name:                                                  sample-app
+Namespace:                                             abc123-prod
+Labels:                                                app=sample-app
+Annotations:                                           <none>
+CreationTimestamp:                                     Fri, 26 May 2023 07:02:57 -0700
+Reference:                                             Deployment/sample-app
+Metrics:                                               ( current / target )
+  resource memory on pods:                             1113748821333m / 1500Mi
+  resource cpu on pods  (as a percentage of request):  6% (28m) / 70%
+Min replicas:                                          3
+Max replicas:                                          5
+Deployment pods:                                       3 current / 3 desired
+Conditions:
+  Type            Status  Reason              Message
+  ----            ------  ------              -------
+  AbleToScale     True    ReadyForNewScale    recommended size matches current size
+  ScalingActive   True    ValidMetricFound    the HPA was able to successfully calculate a replica count from memory resource
+  ScalingLimited  False   DesiredWithinRange  the desired count is within the acceptable range
+Events:           <none>
+```
+
+For more information please read the [Automatically scaling pods with the horizontal pod autoscaler](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html) on Red Hat's tech docs.
+
+
+### PDBs - Pod Disruption Budgets
+The maintenance of clusters occurs within regular business hours. It's important to note that the applications on the platform should be able to handle the sequential rotation of worker nodes.
```

Comment:
Is `sequential` the best word here? The order of node reboots is random. Perhaps `serial` would be better?

Reply:
@IanKWatts what do you think? Thank you!

Reply:
I've updated the wording of that sentence.